src/sentry/api/endpoints/project_performance_issue_settings.py

```diff
@@ -8,15 +8,21 @@
 
 MAX_VALUE = 2147483647
 SETTINGS_PROJECT_OPTION_KEY = "sentry:performance_issue_settings"
+ISSUE_PROJECT_OPTION_KEYS = ["performance_issue_creation_enabled_n_plus_one_db"]
 
 
 class ProjectPerformanceIssueSettingsSerializer(serializers.Serializer):
-    n_plus_one_db_detection_rate = serializers.FloatField(required=False, min_value=0, max_value=1)
-    n_plus_one_db_issue_rate = serializers.FloatField(required=False, min_value=0, max_value=1)
-    n_plus_one_db_count = serializers.IntegerField(required=False, min_value=0, max_value=MAX_VALUE)
-    n_plus_one_db_duration_threshold = serializers.IntegerField(
-        required=False, min_value=0, max_value=MAX_VALUE
+    performance_issue_creation_enabled_n_plus_one_db = serializers.BooleanField(required=False)
+
+
+# We can move this into the project options manager if this sticks around.
+def get_project_option_with_default(project: str, option_key: str) -> bool:
```

Comment:
```suggestion
def get_project_option_with_default(project: Project, option_key: str) -> bool:
```

This doesn't seem like a string.